### PR TITLE
Ca responses ctrl

### PR DESCRIPTION
**Pull request by clanderson87**
Displaying collected data from firebase. Simple and text only at the moment, but I've been staving off death and that's Alex's group project anyway.

@jcockhren @AdamSheaffer @BlaiseGratton @jonathanhuffaker @leifehetland

### CODE REVIEW

File: controllers/responseCtrl.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                gcdKeys.forEach(function(key){
+                  //checking is vm.answerObj has a [questionName] property
+                  if(vm.answerObj.hasOwnProperty([questionName]) === false){
+                    //if not, put the value of this key into that property
+                    vm.answerObj[questionName] = grandChildData[key];
+                  } else {
+                    //aliasing the array at the newly minted vm.answerObj[questionName] location so I can use .push without the console bithing me out
+                    var thisAnswerArray = vm.answerObj[questionName];
+                    //pushing the other keys into thisAnswerArray
+                    thisAnswerArray.push(grandChildData[key].toString());
```
Comment (BlaiseGratton) [inline controllers/responseCtrl.js]: I like your naming/commenting throughout this whole section - if arrays on arrays on arrays is the only way to go, might as well be as neat as possible

**Comment by clanderson87** [comment]
@jonathanhuffaker @leifehetland Thumbs up and merge per favor?

**Comment by jcockhren** [comment]
:+1: 

@clanderson87 Good addition of comments. This will help you and your team out in the long run.

**Comment by leifehetland** [comment]
:+1:

**Pull request merged**